libraries/abieos/test/webpack.config.js

```diff
@@ -1,25 +1,25 @@
 // copyright defined in abieos/LICENSE.txt
 
-const path = require('path');
-const nodeExternals = require('webpack-node-externals');
+const path = require("path");
+const nodeExternals = require("webpack-node-externals");
 
 module.exports = {
-    target: 'node',
+    target: "node",
     externals: [nodeExternals()],
     entry: {
-        test: './src/test.ts',
+        test: "./src/test.ts",
     },
-    mode: 'development',
+    mode: "development",
     module: {
         rules: [
             {
                 test: /\.tsx?$/,
-                use: 'ts-loader',
-                exclude: /node_modules/
-            }
-        ]
+                use: "ts-loader",
+                exclude: /node_modules/,
+            },
+        ],
     },
     resolve: {
-        extensions: ['.tsx', '.ts', '.js']
+        extensions: [".tsx", ".ts", ".js"],
     },
 };
```

Comment:
update code factor to accept single quote

Reply:
this is done by configuring the eslint and prettier configurations